src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,28 @@
 import React from 'react';
+import { Comment } from '../../types/types';
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+type Props = {
+  list: Comment[];
+};
+
+export const CommentList: React.FC<Props> = ({ list }) => {
+  if (list.length > 0) {
+    return (
+      <ul className="CommentList">
+        {list.map(
+          comment => (
+            <li className="CommentItems">
```

Comment:
The 'key' prop should be placed on the 'li' element instead of the 'CommentInfo' component. This helps React optimize the rendering process. 🚀